File: C#/diagonalDifference.cs
public static int diagonalDifference(List<List<int>> arr)
{
    int xySize = arr.Count;
    int rightToLeftSum = 0;
    int leftToRightSum = 0;
    for(int i = 0; i < xySize; i++){
        leftToRightSum += arr[i][i];
        rightToLeftSum += arr[xySize - 1 - i][i];
    }
    return Math.Abs(rightToLeftSum - leftToRightSum);
}

File: Python/DiagonalDifference.py
def diagonalDifference(arr):
    # Write your code here
    rightToLeftSum = 0
    leftToRightSum = 0
    size = len(arr)
    for i in range(size):
        leftToRightSum = leftToRightSum + arr[i][i]
        rightToLeftSum = rightToLeftSum + arr[size - 1 - i][i]
    return abs(leftToRightSum - rightToLeftSum)